ansible/roles/stack-sunbird/templates/content-service_application.conf

```diff
@@ -485,6 +485,7 @@ cloud_storage_type: "{{ cloud_service_provider }}"
 cloud_storage_key: "{{ cloud_public_storage_accountname }}"
 cloud_storage_secret: "{{ cloud_public_storage_secret }}"
 cloud_storage_endpoint: "{{ cloud_public_storage_endpoint }}"
+cloud_storage_proxy_host: "{{ cloud_public_storage_proxy }}"
```

Comment:
@ddevadat  can you please define default value for cloud_public_storage_proxy at ansible/inventory/env/group_vars/all.yml
cc: @santhosh-tg

Reply:
added

Reply:
added default value